Bottom: 9418ef9fedfe2d2ebd24943e011b751ef0bacb89
Top:    a55beef151ff1afe46ddb45fb4beaae38f67b6d8
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2008-10-02 18:32:34 +0100

GTK HID: Raise already open command window to top with ":" key

Previously, ":" only opened the window, but would not raise it
if already open.



---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index 568d46e..09d8025 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -1758,7 +1758,7 @@ LibraryChanged (int argc, char **argv, int x, int y)
 static int
 Command (int argc, char **argv, int x, int y)
 {
-  ghid_handle_user_command (FALSE);
+  ghid_handle_user_command (TRUE);
   return 0;
 }
